.travis.yml

```diff
@@ -1,6 +1,23 @@
-language: node_js
-node_js:
-  - "0.10"
-  - "0.11"
-  - "0.12"
-  - "4.2.4"
+language: c++
+env:
+  - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
+  - NODE_VERSION="0.12"
+  - NODE_VERSION="4.2.4"
+  - NODE_VERSION="5.0"
```

Comment:
This should just be `5` (The current version of node is `5.4.1`, and that won't match `5.0`.